Add note about operator case in Sensu Enterprise Dashbaord Collections guide (#1021)
sensu/sensu-docs: content/sensu-enterprise-dashboard/2.12/collections.md

multiple operators within a single query. Be aware that the use of many
operators may cause higher than normal resource usage.

> _NOTE: When using Operators, they must have all letters capitalized. Operators with any lowercase letter are not supported._

Reviewer: ```suggestion
_NOTE: When using operators, all letters must be capitalized. Operators in lowercase letters are not supported._
```
